test/extended/builds/pipeline_origin_bld.go

```diff
@@ -271,7 +265,9 @@ var _ = g.Describe("[sig-builds][Feature:Builds][sig-devex][Feature:Jenkins][Slo
 				o.Expect(err).NotTo(o.HaveOccurred())
 				err = oc.Run("delete").Args("template", "mongodb-ephemeral").Execute()
 				o.Expect(err).NotTo(o.HaveOccurred())
-				err = oc.Run("delete").Args("secret", "mongodb").Execute()
+				err = oc.Run("delete").Args("secret", "mongodb", "--ignore-not-found=true").Execute()
+				o.Expect(err).NotTo(o.HaveOccurred())
+				err = oc.Run("delete").Args("service", "mongodb", "--ignore-not-found=true").Execute()
```

Comment:
Is this a copy/paste error or did you mean for it to happen twice?

Reply:
intentional .... the template creates both a secret and service called mongodb

Reply:
Ha, i swear i read that 10 times and didn't see that one was secret and one was service :(

Reply:
lol